src/components/Loader/Loader.css

```
@@ -0,0 +1,72 @@
+.spinner-box {
+  --loader-size: 200px;
+  --blue-orbit-size: calc(var(--loader-size) / 2);
+  --green-orbit-size: calc(var(--loader-size) / 2.5);
+  --red-orbit-size: calc(var(--loader-size) / 3);
+  --white-orbit-size: calc(var(--loader-size) / 5);
+
+  width: var(--loader-size);
+  height: var(--loader-size);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: transparent;
+}
+
+.orbit {
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+}
+
+.blue-orbit {
```

❗️
Где-то классы появились в camelCase, где-то остались в kebab-case. Надо прийти к чему-то одному. Сделать это нужно обязательно, но в этом пре можно скипнуть, чтобы не холдить другие таски

И кажется, что лучше было бы
```suggestion
.orbit-blue { // или .orbitBlue, в зависимости от выбранного кейса
```

---

исправил частично, но в следующем ПР приведу все имена классов к camelCase